src/levanter/data/audio.py

```diff
@@ -78,20 +77,25 @@ class BatchAudioProcessor(BatchProcessor[Tuple[np.ndarray, int, str]]):
     def __init__(
         self,
         processor: ProcessorMixin,
+        tokenizer: PreTrainedTokenizerBase,
+        enforce_bos=True,
         enforce_eos=True,
         *,
         batch_size=128,
         override_resources=None,
+        max_length=448,
```

Comment:
that's a strange value!

Reply:
It's the og Whisper's default - I also don't quite understand why? Another annoying thing from the og Whisper is that it uses an encoder size that isn't divisible by 2, so flash attention doesn't work 😂

Reply:
wow

Reply:
![Screenshot_20240315-165510.png](https://github.com/stanford-crfm/levanter/assets/9847335/1a96ccdb-dc26-4806-94c4-4c48abe50ad6)

Strange values galore